### PR TITLE
Updated EventSender and HttpPostImp classes

### PR DESCRIPTION
**Pull request by chillaq**

### CODE REVIEW

File: client/src/main/java/io/split/client/events/EventsSender.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import io.split.client.dtos.Event;
 import io.split.client.utils.Utils;
 import io.split.service.HttpPostImp;
+import io.split.service.SplitHttpClient;
 import io.split.telemetry.domain.enums.HttpParamsWrapper;
 import io.split.telemetry.storage.TelemetryRuntimeProducer;
 import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
```
Comment (nmayorsplit) [inline client/src/main/java/io/split/client/events/EventsSender.java:10]: Please, remove this import that we aren't using anymore.

File: client/src/main/java/io/split/service/HttpPostImp.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package io.split.service;
 
+import io.split.client.dtos.SplitHttpResponse;
 import io.split.client.utils.Utils;
 import io.split.telemetry.domain.enums.HttpParamsWrapper;
 import io.split.telemetry.storage.TelemetryRuntimeProducer;
```
Comment (nmayorsplit) [inline client/src/main/java/io/split/service/HttpPostImp.java:6]: Also here, remove CloseableHttpClient and CloseableHttpResponse imports.

File: client/src/main/java/io/split/service/SplitHttpClientImpl.java [hunk 3]
```diff
@@ -33,7 +33,7 @@ public static SplitHttpClientImpl create(
         return new SplitHttpClientImpl(client, requestDecorator);
     }
 
-    private SplitHttpClientImpl
+    public SplitHttpClientImpl
```
Comment (nmayorsplit) [inline client/src/main/java/io/split/service/SplitHttpClientImpl.java:36]: Why did you make this method public? You can use the create method.

**Review by nmayorsplit — CHANGES_REQUESTED** [review]

File: client/src/main/java/io/split/client/events/EventsSender.java (hunk 1 above)
Comment (chillaq) [reply client/src/main/java/io/split/client/events/EventsSender.java:10]: will do

File: client/src/main/java/io/split/service/SplitHttpClientImpl.java (hunk 3 above)
Comment (chillaq) [reply client/src/main/java/io/split/service/SplitHttpClientImpl.java:36]: Some tests create the httpclient outside methods, which require try catch block, so I made it public, I will updated these tests and revert back to private.

**Pull request closed without merging**